native/machine-coding/blobmaker/app.js

```diff
@@ -11,13 +11,13 @@ let inputs = document.querySelectorAll("input[type='number']");
     });
 
 function createBlob(){
-    let radiusOne = sliders[0].value;
-    let radiusTwo = sliders[1].value;
-    let radiusThree = sliders[2].value;
-    let radiusFour = sliders[3].value;
+    const radiusOne = sliders[0].value;
+    const radiusTwo = sliders[1].value;
+    const radiusThree = sliders[2].value;
+    const radiusFour = sliders[3].value;
 
-    let blobHeight = inputs[0].value;
-    let blobWidth = inputs[1].value;
+    const blobHeight = inputs[0].value;
+    const blobWidth = inputs[1].value;
 
     let borderRadius = `${radiusOne}% ${100-radiusOne}% ${100 - radiusThree}% ${radiusThree}% / ${radiusFour}% ${radiusTwo}% ${100 - radiusTwo}% ${100 - radiusFour}%`;
```

Comment:
Please explain the logic behind this line either in comments or a `README.md` file if required.
Also, you can use const here too.

Reply:
added bruoh

Reply:
Add the explanation of logic too please. The `borderRadius` logic is not readily understandable. So it needs an explanation.